app/scripts/directives/blockEditor.js

```diff
@@ -116,12 +116,17 @@ angular.module('confRegistrationWebApp')
           if (angular.isUndefined($scope.block.content)) {
             $scope.block.content = { 'choices': [] };
           } else if(angular.isUndefined($scope.block.content.choices)){
-            $scope.block.content.choices = [];
+              $scope.block.content.choices = [];
+          }
+          if (_.some($scope.block.content.choices, ['value', newOption])){
+            modalMessage.error('Error: You may not use the same answer more than once for this question. Each answer needs to be unique.');
+          }
+          else {
+            $scope.block.content.choices.push({
+              value: newOption,
+              desc: ''
+            });
```

Comment:
`$scope.editBlockAddOptionValue='';` needs to be added back after this line so when you successfully create a new answer, it clears the new answer input.